### PR TITLE
Add cmake functions for fuzzing and perf tests

### PR DESCRIPTION
**Pull request by ligurio**

### CODE REVIEW

File: perf/CMakeLists.txt [hunk 1]
```diff
@@ -11,10 +11,25 @@ include_directories(${PROJECT_SOURCE_DIR}/src/box)
...
-target_link_libraries(tuple.perftest core box tuple benchmark::benchmark)
+function(create_perf_test)
+  cmake_parse_arguments(
+    PERF
+    ""
+    "PREFIX"
+    "SOURCES;LIBRARIES"
+    ${ARGN}
+  )
+  message(STATUS "Creating performance test ${PERF_PREFIX}_perftest")
```
Comment (ylobankov) [inline perf/CMakeLists.txt:22]: Looks like `${PERF_PREFIX}_perftest` -> `${PERF_PREFIX}.perftest`.

File: test/fuzz/CMakeLists.txt [hunk 2]
```diff
@@ -28,40 +28,63 @@ target_link_libraries(
...
+  )
+  message(STATUS "Creating fuzz test ${FUZZ_PREFIX}_fuzzer")
+  add_executable(${FUZZ_PREFIX}_fuzzer ${FUZZ_SOURCES})
+  target_link_libraries(${FUZZ_PREFIX}_fuzzer PUBLIC ${FUZZ_LIBRARIES})
+  set(FUZZ_TEST_TARGETS "${FUZZ_TEST_TARGETS} ${FUZZ_PREFIX}_fuzzer")
+endfunction()
 
-add_executable(http_parser_fuzzer http_parser_fuzzer.c)
-target_link_libraries(http_parser_fuzzer PUBLIC http_parser fuzzer_config)
+# Use PUBLIC to force 'fuzzer_config' for all dependent targets.
```
Comment (ylobankov) [inline test/fuzz/CMakeLists.txt:47]: Shouldn't this comment be before L43?

**Review by ylobankov — APPROVED** [review]
LGTM

**Review by igormunkin — APPROVED** [review]
@ligurio, nicely done, thanks! LGTM.

**Pull request merged**